front-end/src/components/player-page/stats.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import {
+    Box,
+    FormControl,
+    Grid,
+    InputLabel,
+    MenuItem,
+    Select,
+    SelectChangeEvent,
+    Theme,
+    Typography
+} from "@mui/material";
```

Comment:
Thx for nice spacing here. I see I have a crappy spacing example from my code above lol.